yarn.lock

```diff
@@ -17251,7 +17251,7 @@ folktale@2.3.2:
   resolved "https://registry.yarnpkg.com/folktale/-/folktale-2.3.2.tgz#38231b039e5ef36989920cbf805bf6b227bf4fd4"
   integrity sha512-+8GbtQBwEqutP0v3uajDDoN64K2ehmHd0cjlghhxh0WpcfPzAIjPA03e1VvHlxL02FVGR0A6lwXsNQKn3H1RNQ==
 
-follow-redirects@^1.0.0, follow-redirects@^1.15.4:
+follow-redirects@^1.0.0, follow-redirects@^1.15.0:
   version "1.15.4"
```

Comment:
Should this be `1.15.4`?

Reply:
Do reverts provide a "paper trail" in GH? If so, I was going to revert the follow-redirects upgrade separately afterward. If not, then combining it all in one PR works for me.

Reply:
I think it is just a "new" PR that reverses the commit, but Im not 100%. Im fine either way.